attic/toil-sort-example.py

```diff
@@ -163,11 +154,17 @@ def main():
     if int(options.N) <= 0:
         raise RuntimeError("Invalid value of N: %s" % options.N)
 
-    make_file_to_sort(file_name='file_to_sort.txt', lines=options.num_lines, line_length=options.line_length)
-
-    # Now we are ready to run
-    Job.Runner.startToil(Job.wrapJobFn(setup, os.path.abspath('file_to_sort.txt'), int(options.N), False,
-                                       memory='1000M'), options)
+    file_name = 'file_to_sort.txt'
+    make_file_to_sort(file_name=file_name, lines=options.num_lines, line_length=options.line_length)
+
+    with Toil(options) as toil:
+        if not toil.options.restart:
+            sort_file_url = 'file://' + os.path.abspath('file_to_sort.txt')
+            sort_file_id = toil.importFile(sort_file_url)
+            sorted_file_id = toil.start(Job.wrapJobFn(setup, sort_file_id, int(options.N), False, memory='1000M'))
+            toil.exportFile(sorted_file_id, sort_file_url)
```

Comment:
See comment below: this export should be run in both the restart and non-restart code paths. Otherwise the file isn't written if the workflow fails once and is resumed successfully.